lib/AI-Agent.xml

```diff
@@ -18,11 +18,6 @@
         <Class name="uk.gov.hmcts.reform.sendletter.services.encryption.PgpEncryptionUtil">
```

Comment:
Are any of these being tracked automatically or can we get rid of this file?

Reply:
Encryption and pdf creator tracks - no problem. for some reason a mixture of void methods are not working. Next would be task scheduler + silencing info logs. Only need to know single hit of the task and it's success rate